guides/ManualLabellingSteps.md

```diff
@@ -26,7 +26,8 @@ Below we outline the steps on how to add annotations to these extracted frames.
 
 ### 1. Download and launch the VIA annotation tool
 
-- Access the online version of the tool [here](https://www.robots.ox.ac.uk/~vgg/software/via/via.html) or download it locally [via this link](https://www.robots.ox.ac.uk/~vgg/software/via/) (click on `Downloads` > `Image annotator` section and download the zip file).
+- We use the VIA image annotator tool version 2.0.12.
+- We recommend downloading it locally from [this link](https://www.robots.ox.ac.uk/~vgg/software/via/downloads/via-2.0.12.zip) (or from [here](https://www.robots.ox.ac.uk/~vgg/software/via/), under `Downloads` > `Version 2` > `via-2.0.12.zip`)
```

Comment:
Maybe we can remove the online version for now as it is different interface. Just use the download one

Reply:
I replaced html link with the home page for VIA